src/components/molecules/SettingsLayout/SettingsLayout.styled.tsx

```diff
@@ -14,4 +14,5 @@ export const StyledSettingsContainer = styled.div`
 
   display: flex;
   flex-direction: row;
+  gap: 20px;
```

Comment:
Is that expected? Have you tested it for all pages on both Cloud and OSS?

Reply:
I just changed `padding-right: 20px` to `gap: 20px` as for me it seems more consistant. Did not affect anything.